testsuite/MDAnalysisTests/lib/test_cutil.py

```diff
@@ -46,6 +46,43 @@ def test_unique_int_1d(values):
     assert res.dtype == ref.dtype
 
 
+@pytest.mark.parametrize('arrays', (
+    [],  # empty array
+    [1, 1, 1, 1, ],  # all identical
+    [2, 3, 5, 7, ],  # unique, sorted
+    [5, 2, 7, 3, ],  # unique, unsorted
+    [1, 2, 2, 4, 4, 6, ],  # duplicates, sorted
+    [1, 2, 2, 6, 4, 4, ],  # duplicates, unsorted
+))
+def test_inverse_unique_contiguous_1d_array(arrays):
```

Comment:
`arrays` implies multiple array, but as far as I can tell this is only the one array -- could you please rename this variable for readability?